core/test/unit/middleware/decide-is-admin_spec.js

```diff
@@ -0,0 +1,58 @@
+/*globals describe, beforeEach, afterEach, it*/
+/*jshint expr:true*/
+var sinon           = require('sinon'),
+    decideIsAdmin   = require('../../../server/middleware/decide-is-admin');
+
+describe('Middleware: decideIsAdmin', function () {
+    var sandbox,
+        res,
+        req,
+        next;
+
+    beforeEach(function () {
+        sandbox = sinon.sandbox.create();
+
+        next = sinon.spy();
+        res = sinon.spy();
+        req = {};
+    });
+
+    afterEach(function () {
+        sandbox.restore();
+    });
+
+    it('sets the isAdmin flag if the url contains /ghost/.', function (done) {
```

Comment:
This was my morning entertainment btw - thanks for taking the time to link to slack :+1: